server/cluster/cluster.go

```diff
@@ -43,6 +47,10 @@ func (c *Cluster) Name() string {
 func (c *Cluster) Load(ctx context.Context) error {
 	c.lock.Lock()
 	defer c.lock.Unlock()
+	if c.coordinator == nil {
```

Comment:
I guess the method name should be renamed to `Init` rather than `Load`.

Reply:
#29  will fix this